jsonmatch/matcher.py

```diff
@@ -224,9 +233,11 @@ def __init__(self, spec, against):
     def add_break(self, path_tuple, spec_val, against_val):
         self.paths_to_breaks[path_tuple] = (spec_val, against_val)
 
-    def __nonzero__(self):
+    def __bool__(self):
         return bool(self.paths_to_breaks)
 
+    __nonzero__ = __bool__
```

Comment:
if you `from builtins import object` you don't need to define `__nonzero__`

Reply:
Updated